scripts/test_emscripten.sh

```diff
@@ -39,6 +39,4 @@ SOLJSON="$REPO_ROOT/$BUILD_DIR/libsolc/soljson.js"
 VERSION=$("$REPO_ROOT"/scripts/get_version.sh)
 
 echo "Running solcjs tests...."
-"$REPO_ROOT/test/solcjsTests.sh" "$SOLJSON" "$VERSION"
-echo "Running external tests...."
-"$REPO_ROOT/test/externalTests.sh" "$SOLJSON"
```

Comment:
`externalTests.sh` is not called anymore within CI and exists for convenience. Also fine with removing it.

Reply:
Wait, but don't we lose the other external tests (besides `solc-js`) with this change?

Reply:
Or did this use to run only `solc-js`?

Reply:
The thing is, that we ran the external tests on every build *and* on the nightly build (by calling the project-specific script separately). This removes the runs on PRs, but doesn't touch the nightly ones. And for `solc-js` it's only the path to the script that changes.

Reply:
Which in turn brings down build times.